src/Components/Cart/CartProduct.js

```diff
@@ -0,0 +1,44 @@
+import { useCart } from "../../Context/cart-context"
+
+
+function CartProduct({product, discount}) {
+    const { dispatch} = useCart();
+    return (
+         <div className="cart-item flex-row margin-tb">
+                            <div className="cart-item-image-container">
+                                <img className="cart-item-image" src="/img/pexels-ron-lach-10397680.jpg" alt="shirt" />
+                            </div>
+                            <div className="cart-item-desc-container">
+                                <div className="cart-item-desc">
+                                    <h3>{product.title}</h3>
+                                    <h3>${product.price}</h3>
```

Comment:
Can destructure product at top of file, won't have to prefix product. before everything

Reply:
Actually the product object is required to be sent through dispatch which is why didn't destructure it